package.json

```diff
@@ -4,7 +4,7 @@
         "type": "ssh",
         "url": "https://github.com/guardian/atoms-rendering"
     },
-    "version": "10.0.2",
+    "version": "11.0.0-0",
```

Comment:
I think just 11.0.0

Reply:
That's come from the pre-major release using yarn release. I assume that if I merge to main and then release it as a major release that should fix it?

Reply:
You'll then have to PR the change again though, so your best bet is to set it here and then ensure you release as 11.0.0 (I think that's correct @oliverlloyd ?)

Reply:
So actually you never want to have a PR with changes to `version` in it.

If you're releasing a new version in the normal course of things (non pre release) then it's done directly on `main` so there's no PR

If you're releasing a pre release, like here, then that _can_ be done on the branch (using `--any-branch`) but you don't need to carry that `version` diff into main. You can delete this diff and then when you do the main release on `main` with `yarn release` it will bump it as required.

Reply:
Right, perfect, thanks!